### PR TITLE
Create si.lua

### PR DESCRIPTION
**Pull request by PandaSLO**

### CODE REVIEW

File: data/locales/si.lua [hunk 1]
```diff
@@ -0,0 +1,86 @@
+return {
+-- Buttons, context menu
+['ui_use'] = "Uporaba",
+['ui_give'] = "Daj",
+['ui_close'] = "Zapri",
+['ui_drop'] = "Spusti",
+['ui_removeattachments'] = "Odstrani priloge",
+-- Tooltip
+['ui_durability'] = "Trajnost",
+['ui_ammo'] = "strelivo",
```
Comment (LukeWasTakenn) [inline data/locales/si.lua:10]: Should this be lowercase?

File: data/locales/si.lua [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+['weapon_hand_wrong'] = "Napačno orožje v roki",
+['open_player_inventory'] = "Odpri inventar igralca~",
+['open_secondary_inventory'] = "Odpri sekundarni inventar~",
+['disable_hotbar'] = "Prikaži vročo vrstico inventarja~",
+['reload_weapon'] = "Znova naloži orožje~",
+['use_hotbar'] = "Uporabi element vroče vrstice %s~",
+['no_durability'] = "Trajnost artikla je izčrpana",
+['cannot_give'] = "Cilju ni mogoče dati %s %s",
+['evidence_cannot_take'] = "Ni dovolj visoka ocena, da bi jemljejo iz dokazov",
+['dumpster'] = "smetnjak"
```
Comment (LukeWasTakenn) [inline data/locales/si.lua:85]: Should this be lowercase as well?

File: data/locales/si.lua (hunk 1 above)
Comment (PandaSLO) [reply data/locales/si.lua:10]: yes

Comment (PandaSLO) [reply data/locales/si.lua:10]: in our language we would type that in lowercase

File: data/locales/si.lua (hunk 2 above)
Comment (PandaSLO) [reply data/locales/si.lua:85]: yes

File: data/locales/si.lua (hunk 1 above)
Comment (LukeWasTakenn) [reply data/locales/si.lua:10]: I have no idea how that makes any sense, but cool!

File: data/locales/si.lua (hunk 2 above)
Comment (SloRunner) [reply data/locales/si.lua:85]: evidence_cannot_take is not right btw

File: data/locales/si.lua (hunk 1 above)
Comment (SloRunner) [reply data/locales/si.lua:10]: ui_removeattachments is not right also

**Pull request merged**